components/script/webdriver_handlers.rs

```diff
@@ -1706,3 +1708,20 @@ pub(crate) fn handle_is_selected(
         )
         .unwrap();
 }
+
+pub(crate) fn handle_try_wait_for_document_navigation(
```

Comment:
This function doesn't perform any wait but simply returns the document ready status.

How/where is the "wait" performed?